### DISCUSSION TITLE
Does velero support my cluster? What provider should I use? ...

### DISCUSSION THREAD

**Discussion by arisjr** [vmware-tanzu/velero#3739] [Community support Q&A] [answered]
Hello for you all.

I'm new to velero, and I'm a local kubernetes cluster admin.

I have an old (v. 1.14.10) kubernetes local bare metal installation. I use it with an cephfs cluster file system for persistent volumes. I have installed it with the oficial kubernetes kubeadm. Now I need to update and add more masters on the cluster, and the need to backup is obvious. I need to backup it all, deployments, DS, StatefulSet, secrets, etc... I don't need to backup the volumes, since it's dealt by ceph separately, and we have Petabytes of storage. 

https://velero.io/docs/v1.6/supported-providers/
I don't know how to categorize my cluster.
Does Velero support my type of cluster? 
What is the type of provider I need to use with velero? 
And what version of velero should I use that fits best with my k8s version?

Best regards,

**Comment by jenting** [comment, marked as answer]
1. If I remember correctly, Velero supports K8s >= 1.12
2. Since you don't need to backup the volumes, the plugin depends on which S3 provider you want to choose.
- AWS: velero-plugin-for-aws
- GCP: velero-plugin-for-gcp
- Azure: velero-plugin-for-microsoft-azure
- S3-compatible (like MinIO): velero-plugin-for-aws

**Reply by arisjr** [reply]
If I understand correctly, I will have to obligatory use a S3 provider (ceph rados gateway on my case). There's no chance to backup the k8s resources on a normal volume... or there is a chance?

**Reply by jenting** [reply]
No, Velero _only_ supports S3 object storage for now.

**Reply by arisjr** [reply]
Thanks @jenting, I already made my first backup on the ceph rados gateway! Best regards!
